profiles/api_test.py

```diff
@@ -235,6 +242,35 @@ def test_parse_import_alumni_csv():
     ]
 
 
+def test_parse_import_alumni_csv_case_insensitive(tmp_path):
+    """parse_alumni_csv should import the user email case insensitive"""
+    path = tmp_path / "test.csv"
+    with open(path, "w") as f:  # create file
+        f.write(
+            f"{ALUM_LEARNER_EMAIL},{ALUM_BOOTCAMP_NAME},{ALUM_BOOTCAMP_RUN_TITLE},{ALUM_BOOTCAMP_START_DATE},{ALUM_BOOTCAMP_END_DATE}\n"
+        )
+        f.write("HDOOF@ODL.MIT.EDU,,,,\n")
+        f.write("EXAMPLE@ODL.MIT.EDU,,,,\n")
```

Comment:
It would be better to use non-sendable email addresses:
```suggestion
        f.write("HDOOF@EXAMPLE.COM,,,,\n")
        f.write("EXAMPLE@EXAMPLE.COM,,,,\n")
```